tests/test_ofc.py

```diff
@@ -127,20 +127,7 @@ def test_set_pssn_gain(self):
 
     def test_calculate_corrections(self):
 
-        gain = 1.0
-        filter_name = ""
-        rot = 0.0
-
-        wfe, field_idx = self._get_wfe()
-
-        (
-            m2_hex_corr,
-            cam_hex_corr,
-            m1m3_corr,
-            m2_corr,
-        ) = self.ofc.calculate_corrections(
-            wfe=wfe, field_idx=field_idx, filter_name=filter_name, gain=gain, rot=rot
-        )
+        m2_hex_corr, cam_hex_corr, m1m3_corr, m2_corr = self._calculate_corrections()
```

Comment:
I am just going to point out that this method is called only once in the entire code base.. I honestly don't think it is worth breaking it down this way, it mades it harder to understand what is happening here, not easier...

Reply:
This `self._calculate_corrections()` was called more than once actually. But I removed this and put it back if this is what you want. The update is in commit: e64d8ae. Thanks!